### ISSUE TITLE
Resuming training from given 100k checkpoint collapses

### ISSUE THREAD

**Issue by isaac-dunn** [ajbrock/BigGAN-PyTorch#26]
Training on a single V100 GPU, resuming from the checkpoint at 100,000 iterations linked to from the README, consistently collapses straight away.

Screenshot of script to resume training from checkpoint: https://drive.google.com/open?id=1ik16IVE9G8l7dCnpCjQStVslkE9Ltg19

The differences I can see besides using just one GPU and so having a smaller batch size and more accumulations are:

- not using parallel
- using 0 workers
- not using multiepoch sampler
- not using the EMA for evaluation (to show collapse)

I'm very confused as to why this is not working - none of those changes should make a difference, as far as I can tell - any suggestions greatly appreciated!

GIF showing collapse over about 130 iterations, sampling every 5: https://giphy.com/gifs/ifMnHURSReyjyE3IfC/html5

Logs: https://drive.google.com/drive/folders/11Far9osGQ2KslKY4Gyio8AHotBGXTKI6?usp=sharing

**Comment by isaac-dunn** [comment]
I've realised what the problem was. The directories for the images of different classes were saved in `data/ImageNet/train`, whereas they should have been placed in `data/ImageNet`. Because labels are assigned based on which subdirectory of `ImageNet` the images occupy, they were all being given the same label, `0`.

Whoops!
